app/src/main/java/com/example/plantsapp/presentation/ui/tasksfordays/TasksForDaysFragment.kt

```diff
@@ -33,6 +34,7 @@ class TasksForDaysFragment : Fragment(R.layout.fragment_tasks_for_days) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        (activity as AppCompatActivity).supportActionBar?.show()
```

Comment:
But what will happen if this fragment will not be our start fragment? (For example if default fragment will be Profile)

Remove this from here